extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -289,7 +294,7 @@ export class ImageViewer {
     // and then naturally upgrade to a higher quality image.
     return timerFor(this.win).promise(1).then(() => {
       this.image_.setAttribute('src', src);
-      return this.loadPromise(this.image_);
```

Comment:
if we import it directly that will mean the `loadPromise` implementation as well as its dependency inside that module will then be compiled into the `amp-image-lightbox.js` binary. The reason the delegation was done was so that the `loadPromise` code and `event-helper.js` code is only inside the `v0.js` binary and not inside extensions.